resources/definitions/atomPlus.def.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Atom Plus",
```

Comment:
I've been unable to find any information on this printer online to verify its dimensions and such. Do you know of any source of information about this printer?

Reply:
The Plus model was never publicly released, we obtained it as testers, I believe it's only available as a custom build.